packages/lab/src/__tests__/__e2e__/number-input/NumberInput.cy.tsx

```diff
@@ -194,6 +194,34 @@ describe("Number Input", () => {
     );
   });
 
+  it("calls the `onChangeEnd` callback with the final value after continuous increments/decrements", () => {
+    const changeEndSpy = cy.stub().as("changeEndSpy");
+    cy.mount(<Default onChangeEnd={changeEndSpy} />);
+    cy.findByLabelText("increment value")
+      .realMouseDown()
+      .wait(1000)
+      .realMouseUp();
+
+    cy.get("@changeEndSpy").should(
+      "have.been.calledOnceWith",
+      Cypress.sinon.match.any,
+      7,
```

Comment:
curious - how do we know the value will be 7 after  1 second..

probably missed something.